api/composer.json

```diff
@@ -1,12 +1,12 @@
 {
     "minimum-stability": "dev",
     "require": {
-        "craftcms/cms": "3.7.52",
-        "craftcms/google-cloud": "1.4.1",
-        "craftcms/redactor": "2.10.10",
-        "sebastianlenz/linkfield": "2.0.0-rc.2",
-        "spicyweb/craft-neo": "2.13.15",
-        "vlucas/phpdotenv": "^3.4.0"
+        "craftcms/cms": "~4.15.0.2",
+        "craftcms/google-cloud": ">2.2.0",
```

Comment:
Now that we were able to update Craft and the plugins to their compatible versions, please change to the prefix for this to `^`